assign default encryption scheme for passphrase with none.

KtorZ opened the pull request:
# Issue Number

<!-- Put here a reference to the issue this PR relates to and which requirements it tackles -->

#1701 

# Overview

<!-- Detail in a few bullet points the work accomplished in this PR -->

- a4dd767006047e0fc14fb6de7b20920ae854dc03
  :round_pushpin: **assign default encryption scheme for passphrase with none.**


# Comments

<!-- Additional comments or screenshots to attach if any -->

cc @nikolaglumac

<!-- 
Don't forget to:

 ✓ Self-review your changes to make sure nothing unexpected slipped through
 ✓ Assign yourself to the PR
 ✓ Assign one or several reviewer(s)
 ✓ Once created, link this PR to its corresponding ticket
 ✓ Assign the PR to a corresponding milestone
 ✓ Acknowledge any changes required to the Wiki
-->

KtorZ committed:
assign default encryption scheme for passphrase with none.

paweljakubas reviewed (approved):
looks like a fix that addresses the case when we had updated password that had not passphrase schema  and was not exposed (because resulted in Nothing)   

Alternatively, we could solve this via manual migration...

KtorZ committed:
remove outer border in table being interpreted as docstrings...

KtorZ committed:
add manual migration for wallet passphrase scheme

This is cleaner than doing some add-hoc transformation when the wallet
gets read from the database. And, it allows for some unit testing to
happen!

KtorZ committed:
add regression test to show manual migration of password scheme is working as intended

KtorZ commented on lib/core/src/Cardano/Wallet/DB/Sqlite.hs:
```diff
@@ -378,6 +414,7 @@ migrateManually tr defaultFieldValues =
                 _ <- Sqlite.step addColumn
                 Sqlite.finalize addColumn
       where
+        activeSlotCoeff = DBField CheckpointActiveSlotCoeff
```
moved from above.

KtorZ commented on lib/core/src/Cardano/Wallet/DB/Sqlite.hs:
```diff
@@ -353,6 +388,7 @@ migrateManually tr defaultFieldValues =
                 Sqlite.finalize stmt
       where
         hardLowerBound = toText $ fromEnum $ minBound @(Index 'Hardened _)
+        rndAccountIx   = DBField RndStateAddressAccountIndex
```
moved from above.
